mainwindow.cpp

```diff
@@ -64,10 +64,8 @@ MainWindow::MainWindow(QSettings &settings)
     m_ui->mainToolBar->insertWidget(m_ui->actionOpen_Password, sessionMenuButton);
     */
 
-    // TODO: Could we make connections easier?
     // We connect John process' signals with our slots.
     // John was ended.
-    // TODO: It will good to show exit status and exit code to user.
```

Comment:
I think it's too complex information for the user. Most users won't understand what exit code mean.